setup.cfg

```diff
@@ -39,6 +39,16 @@ install_requires =
 spacy_factories =
     llm = spacy_llm.pipeline.llm:make_llm
 
+[options.extras_require]
+minichain =
+    minichain>=0.3,<0.4
+langchain =
+    langchain>=0.0.144,<0.1
+hf =
+    cupy-cuda11x
+    torch>=1.13.1,<2.0
+    transformers>=4.28.1,<5.0
```

Comment:
To be parallel to the spacy extras, I would call this option `transformers` and leave out `cupy`. (I don't think you need it and if you do, `cupy-cuda11x` is too specific as a version. I can make suggestions if you do need to keep it here.)

Reply:
Good point. I aligned the packages on the README, maybe there's something to change there as well.